flink/flink-common/src/main/java/com/netease/arctic/flink/table/DynamicTableFactory.java

```diff
@@ -124,8 +126,13 @@ public DynamicTableSource createDynamicTableSource(Context context) {
       objectPath = new ObjectPath(identifier.getDatabaseName(), identifier.getObjectName());
     }
 
+    // update computed columns and watermark to properties
+    Map<String, String> properties = options.toMap();
```

Comment:
Based on the implementation of `ArcticCatalog#getTable`, the `CatalogTable#options` already contains the complete properties. What is the reason for generating them again here?

Reply:
> Based on the implementation of `ArcticCatalog#getTable`, the `CatalogTable#options` already contains the complete properties. What is the reason for generating them again here?

Yes, It seems no need to generate properties.